web-app/src/resources/elements/image/image-canvas.js

```diff
@@ -436,6 +436,14 @@ export class ImageCanvas {
                 cvs.attr('width', this._toScaledSize(img.width));
                 cvs.attr('height', this._toScaledSize(img.height));
 
```

Comment:
So this change messes things up for me.  I have uploaded an example image I use.  If you open this it says 100% but it is not.  It is somewhere around 56% or something.  This is because I scan at the preferred 600dpi and the actual image is much larger than my monitor 1 for 1.  As I said before you can't set the width to 100%... that is not how Canvas works.  I am not seeing a problem here without this

![img010](https://user-images.githubusercontent.com/3221886/189968733-d3fd69b0-64b0-438d-96b1-71399fecf365.jpg)

Reply:
I agree with you on this.  I will re-work this.

Reply:
Just curious when you open your image, do you prefer to fit the entire image to your screen or just to the width.

Reply:
If we are going to set that scaling factor is optionally reset to 100% then I would expect it to open at 100%.  If it is bigger than alloted pixels... that is fine.  If it is less... that is fine two.  As long as it is 100% (or whatever I had before I opened image say 25% using the preference)